sippy-ng/src/bugs/BugButton.js

```diff
@@ -16,24 +16,26 @@ export default function BugButton(props) {
 
   const text = `The following test is failing:
   
-  ${props.testName}
+  {code:none}${props.testName}{code:none}
   
 Additional context here:
   
   ${document.location.href}`
```

Comment:
I know you haven't touched this but what do you think about making this a link so the jiras don't have that long url in them?

Reply:
great idea, that bothered me